docs/ref/carbond.limiter.TooBusyLimiter.rst

```diff
@@ -11,18 +11,18 @@ the process is.
 Class
 -----
 
-.. js:class:: carbond.limiter.TooBusyLimiter
+.. js:class:: carbond.limiter.TooBusyLimiter()
 
     *extends*: :js:class:`~carbond.limiter.FunctionLimiter`
 
 Properties
 ----------
 
-.. js:class:: Collection()
+.. js:class:: carbond.limiter.TooBusyLimiter()
     :noindex:
     :hidden:
 
-    .. js:attribute:: absMaxOutstandingReqs
+    .. js:attribute:: carbond.limiter.TooBusyLimiter.absMaxOutstandingReqs
```

Comment:
Full name is specified here because Javascript domain linking is not correctly handled.